scrapy/settings/default_settings.py

```diff
@@ -230,9 +230,12 @@
 STATS_DUMP = True
 
 STATSMAILER_RCPTS = []
-
-TEMPLATES_DIR = abspath(join(dirname(__file__), '..', 'templates'))
-
+TEMPLATES_DIR_BASE = [
```

Comment:
@SudShekhar Since we are deprecating the `_BASE` settings in favor of using priorities on `Settings`, I think we should do the same. But still keep the setting as list.